mlflow/metrics/genai/genai_metric.py

```diff
@@ -284,10 +294,18 @@ def aggregate_function(aggregate_option, scores):
 
         return MetricValue(scores, justifications, aggregate_results)
 
+    signature_parameters = [
+        Parameter("predictions", Parameter.POSITIONAL_OR_KEYWORD, annotation="pd.Series"),
+        Parameter("metrics", Parameter.POSITIONAL_OR_KEYWORD, annotation=Dict[str, MetricValue]),
+        Parameter("input", Parameter.POSITIONAL_OR_KEYWORD, annotation="pd.Series"),
+    ]
```

Comment:
can we read the current signature here? would be easier to not have to maintain this

Reply:
Thought about it, but we would need to pop the *args out of the signature. We can go either way. 
We can update this code if we feel strongly about doing that instead?